matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/TimelineEventDecryptor.kt

```diff
@@ -101,9 +101,7 @@ internal class TimelineEventDecryptor @Inject constructor(
         executor?.execute {
             Realm.getInstance(realmConfiguration).use { realm ->
                 try {
-                    runBlocking {
```

Comment:
if the aim is to ensure the realm instance is only executed on a single thread it might be safer to keep the `runBlocking` here next to the instance creation